cmd/otelauto/main.go

```diff
@@ -33,6 +38,14 @@ func main() {
 		os.Exit(-1)
 	}
 
+	c := make(chan os.Signal, 1)
```

Comment:
To be deleted, as the last version in the `main` branch already implements graceful desktop.